packages/list_bloc/lib/src/cubits/list_cubit.dart

```diff
@@ -20,5 +20,57 @@ class ListCubit<T, F> extends DataCubit<List<T>, F> {
       rethrow;
     }
   }
-// TODO(alexis): implement remove, add, replace etc
+
+  Future<void> remove(T item) async {
```

Comment:
Hi @ChHannan Thank you for the PR. Can you please explain briefly what is happening in this method?

Reply:
The method takes the input of type generic and emit the loading state.  It will load the current data state and check if it’s null then return. Otherwise, it creates a copy of current data list and remove the specified  item. As a result of removal, if the list is empty, emit empty data state and if it's not then emit data state with updated data.